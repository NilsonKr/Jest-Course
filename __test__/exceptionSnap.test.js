describe('Handle exceptions on the snapshots', () => {
    test('Always will failed ', () => {
        const data ={
            date: new Date(),
            id: Math.floor(Math.random() * 21)
        }
        expect(data).toMatchSnapshot('Always failed')
    });

    test('Handle Exceptions', () => {
        const data = {
            createAt: new Date(),
            id: Math.floor(Math.random() * 21),
            name: 'Nilson Diaz'
        }

        expect(data).toMatchSnapshot({
            createAt: expect.any(Date),
            id: expect.any(Number),
        })
    })
});
