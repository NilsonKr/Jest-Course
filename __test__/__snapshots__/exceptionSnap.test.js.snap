// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Handle exceptions on the snapshots Always will failed : Always failed 1`] = `
Object {
  "date": 2021-02-20T06:22:02.466Z,
  "id": 0,
}
`;

exports[`Handle exceptions on the snapshots Handle Exceptions 1`] = `
Object {
  "createAt": Any<Date>,
  "id": Any<Number>,
  "name": "Nilson Diaz",
}
`;
